src/main/java/Duke.java

```diff
@@ -1,10 +1,143 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Sura");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+
+        Scanner scanner = new Scanner(System.in);
+        Task[] tasks = new Task[100];
```

Comment:
Can add named constant MAX_ENTRIES = 100 to avoid magic numbers. Makes code more readable and easier to understand for other developers.